src/StreamJsonRpc.Shared/JsonRpc.cs

```diff
@@ -654,6 +661,38 @@ protected virtual Task<ReturnType> InvokeCoreAsync<ReturnType>(int? id, string t
                 }
             }
         }
+        
+        private static Dictionary<string, List<string>> GetMethodToAttributeMap(object target)
+        {
+            var methodToAttributeMap = new Dictionary<string, List<string>>();
```

Comment:
Can you specify a `StringComparer` argument to the dictionary's ctor? Even if it matches what would be the default behavior, it's good to know that it was considered.